### ISSUE TITLE
Is there a way to have these be campaign specific?

### ISSUE THREAD

**Issue by Avitale34** [earlSt1/vtt-navigation-presets#2]
Is there a way to have these be campaign specific? Instead of rolling over into every campaign.

**Comment by earlSt1** [comment]
Hi, what do you mean by campaign specific? Your defined presets should be unique to each world you use.

**Comment by Avitale34** [comment]
That does not seem to be the case for me, my descent into avernus folders are in my DOTMM world.

**Comment by earlSt1** [comment]
Can you send me your Foundry version, system version, and Navigation Presets version?

**Comment by Avitale34** [comment]
Foundry 0.6.6 DnD 0.96 Nav Preset 1.0.4

**Comment by earlSt1** [comment]
So when you create a preset in world1 it appears in both world1 and world2?

Can you try creating a new world and seeing if it affects that one too?

**Comment by Avitale34** [comment]
Correct, Folders created in world 1 appear in world 2, creating from scratch works as its suppose to with a default blank folder.

**Comment by earlSt1** [comment]
That's unusual. Raises some more questions :) 

Are your worlds named similarly/the same? 
Are you using Forge or some hosting service? 
Did you copy one world to the other when creating it? (eg. copy world1 and rename to world2)
Are you experiencing any other module issues with world settings?
When you installed Navigation Presets were you in the middle of transferring/copying worlds?

The settings for worlds are stored in `Data/<worldName>/data/settings.db`. Are you able to compare the settings files for both worlds?

**Comment by Avitale34** [comment]
Not named similarly
Not using any hosting services just my IP
I did convert both over from Roll20 using kakaroto's exporter
No other issues that I know of
negative no transferring of copying

**Comment by earlSt1** [comment]
Could you send me both `settings.db` files for both your worlds? You can find them in the location in my previous comment.

**Comment by earlSt1** [comment]
Hi, are you still experiencing this issue in 0.8.x?
